Sources/Kanna/libxmlHTMLNode.swift

```diff
@@ -142,6 +142,12 @@ final class libxmlHTMLNode: XMLElement {
         self.weakDocument = document
         self.docPtr       = docPtr
         self.nodePtr      = xmlDocGetRootElement(docPtr)
```

Comment:
This line should be deleted.

Reply:
Which line you meant?

Reply:
"self.nodePtr      = xmlDocGetRootElement(docPtr)" ?

Reply:
I already deleted line 144 in my commit and replaces with if-let-else. No? @tid-kijyun

Reply:
The line has not been deleted in your commit.  It remains on the line above the if-let-else line.
I think this is a simple mistake :)
https://github.com/tid-kijyun/Kanna/blob/e56ee3332de8bc2367a62b56c0c93036c27036f4/Sources/Kanna/libxmlHTMLNode.swift#L144-L149